fastlane/lib/fastlane/actions/gradle.rb

```diff
@@ -68,7 +68,7 @@ def self.run(params)
 
         apk_search_path = File.join(project_dir, '**', 'build', 'outputs', 'apk', '**', '*.apk')
         aab_search_path = File.join(project_dir, '**', 'build', 'outputs', 'bundle', '**', '*.aab')
-        output_json_search_path = File.join(project_dir, '**', 'build', 'outputs', 'apk', '**', 'output.json')
+        output_json_search_path = File.join(project_dir, '**', 'build', 'outputs', 'apk', '**', 'output-metadata.json')
```

Comment:
```suggestion
        output_json_search_path = File.join(project_dir, '**', 'build', 'outputs', 'apk', '**', 'output*.json') # output.json in Android Stuido 3 and output-metadata.json in Android Studio 4
```

Reply:
@mrkazansky This was marked as resolved but not changed 🤷‍♂️ Does this change look okay to you?

Reply:
i think this is ok bro

Reply:
@joshdholtz  I wonder when can this change be merged?